src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx

```diff
@@ -207,16 +222,20 @@ const DateForm = ({
   const textKeys = useTextKeys()
   const { isoLocale, marketLabel } = useCurrentLocale()
 
+  // FIXME: get this data from BE
+  const isCarSwitcher = false
+  const isSwitcher = currentInsurer?.switchable || isCarSwitcher
+
   const [dateValue, setDateValue] = useState(() =>
-    getDefaultDateValue(startDate, currentInsurer),
+    getDefaultDateValue(startDate, isSwitcher),
   )
   const [dateLocale, setDateLocale] = useState<Locale | null>(null)
 
   const [datePickerOpen, setDatePickerOpen] = useState(false)
 
   useEffect(() => {
-    setDateValue(getDefaultDateValue(startDate, currentInsurer))
-  }, [startDate, currentInsurer])
+    setDateValue(getDefaultDateValue(startDate, isSwitcher))
```

Comment:
Nit: dateValue is calculated from props, we never set it other way.  We can probably simplify code by calculating it and removing the state

Reply:
Nice, didn't even notice 👍  Will fix!